v-next/hardhat-ethers/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "@ignored/hardhat-vnext-ethers",
+  "version": "3.0.0-next.2",
+  "description": "Hardhat plugin for ethers",
+  "homepage": "https://github.com/nomicfoundation/hardhat/tree/v-next/v-next/hardhat-ethers",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/NomicFoundation/hardhat",
+    "directory": "v-next/ethers"
+  },
+  "author": "Nomic Foundation",
+  "license": "MIT",
+  "type": "module",
+  "exports": {
+    ".": "./dist/src/index.js"
```

Comment:
I think we should also have an export for the types

Reply:
Yes, good catch. [Done here](https://github.com/NomicFoundation/hardhat/pull/5787/commits/5a170acac329dc58922f1a11a37fa813cc874dcb)